cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -39,7 +39,9 @@ import (
 
 const (
 	// UpgradeManifestTimeout is timeout of upgrading the static pod manifest
-	UpgradeManifestTimeout = 5 * time.Minute
+	UpgradeManifestTimeout   = 5 * time.Minute
+	EtcdUpgradeRetries       = 10
+	EtcdUpgradeRetryInterval = 15 * time.Second
```

Comment:
golint failed:
exported const `EtcdUpgradeRetries` should have comment (or a comment on this block) or be unexported
exported const `EtcdUpgradeRetryInterval` should have comment (or a comment on this block) or be unexported

Reply:
We should probably not export these constants at all. They should start with a lowercase letter.
Also, these should probably be changed to `etcdConnectRetries` and `etcdConnectRetryInterval`.